parsec/backend/events.py

```diff
@@ -133,7 +133,9 @@ def _on_invite_status_changed(
                 client_ctx.logger.warning("dropping event (queue is full)")
 
         def _on_pki_enrollment_updated(
-            event: APIEvent, backend_event: BackendEvent, organization_id: OrganizationID
+            event: APIEvent,
+            backend_event: BackendEvent,
+            organization_id: OrganizationID,
```

Comment:
Why this format ? Is it `black` that do that ?

Reply:
I accidentally ran `black` on non-modified files, so it makes unnecessary changes. I'll revert the changes.